datadoc/backend/DataDocMetadata.py

```diff
@@ -38,21 +40,20 @@
     ]
 )
 
+METADATA_DOCUMENT_FILE_SUFFIX = "__DOC.json"
```

Comment:
Just a personal opinion, but frameworks like Spark will ignore files with a `.`-prefix (like` .metadata.json`) when reading multiple files from a folder. So maybe we should use this as well.

Reply:
Good suggestion, I might defer to @BjornRoarJoneid to see what he thinks about that. I personally don't have anything against it, and it also may make it more clear to users that the file should not be manually edited.